packages/react-dom/src/server/escapeScriptForBrowser.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {checkHtmlStringCoercion} from 'shared/CheckStringCoercion';
+
+const scriptRegex = /(<\/|<)(s)(cript)/gi;
+const scriptReplacer = (match, prefix, s, suffix) =>
+  `${prefix}${substitutions[s]}${suffix}`;
```

Comment:
Let's make this an inline condition instead of going through a whole hidden class check and fake-map look up.

Reply:
I meant `s === ‘s’ ? '\\u0073' : '\\u0053’;

Reply:
👍